src/js/services/txFormatService.js

```diff
@@ -26,7 +26,7 @@ angular.module('copayApp.services').factory('txFormatService', function(profileS
 
     var outputs = lodash.isArray(tx.outputs) ? tx.outputs.length : 0;
```

Comment:
It 's not necessary. The code outside the condition will complete the required info (toAddress, amountStr, AlternativeAmountStr, feeStr).